### ISSUE TITLE
[Bug] Degraded audio quality in VCs on Dorion

### ISSUE THREAD

**Issue by PythonTryHard** [SpikeHD/Dorion#106]
**Describe the bug**
Music from bots played through Dorion sounds worse than official client (major treble loss)

**To Reproduce**
1. Join any VCs with a music bot.
2. Play any song (good headphones is recommended but not strictly necessary). I used [Kobaryo - Bookmaker](https://hitnex.bandcamp.com/track/bookmaker)
3. Compare to audio official client.

**Expected behavior**
Sound quality on par with official client.

**System Info (please complete the following information):**
 - OS: Windows 11 22H2 (build 22621.1992)
 - Version: v1.2.1. Also seen in v1.1.x.
 - Any other relevant info: None

**Additional context**
Audio debug windows cannot be opened on Dorion, making figuring what is wrong difficult.

**Comment by SpikeHD** [comment]
Is this still occurring? I wonder if they just have different quality settings set for the audio streams depending on user agent (which is now set to Chrome in Dorion, since it fixes voice chat on all platforms), which would maybe explain why Firefox sounds fine?

**Comment by PythonTryHard** [comment]
It's still occurring as of client 2.1.0, from a fresh installation (no plugins, no nothing)

**Comment by 1111-glitch** [comment]
Hey just wanted to know if there has been any progress on this? I believe the issue I've been having where my mic sounds noticeably worse on dorion compared to web/desktop is related to this as well.

I'd be happy to provide logs or whatever needed if you let me know what you need.

**Comment by SpikeHD** [comment]
No, I have not spent time looking into this recently, though I did a long time ago without success

**Comment by 1111-glitch** [comment]
Well if you need logs to reproduce I'd be happy to add you on discord to talk through it. I'd really like to switch over to dorion cause its so much more lightweight. 

I do actually have an idea on what the issue might be related to. I use [Equalizer APO](https://sourceforge.net/projects/equalizerapo/) to EQ my mic and desktop audio which hooks directly into a windows audio system. Dorion could be bypassing this somehow because my microphone does actually just sound how it would sound with no EQ at all.

**Comment by SpikeHD** [comment]
I'm afraid its not so much a thing that can be solved with logs, and more how the underlying webview engine works. Somewhere somehow there is a difference in Webview2 that is causing the worse quality (potentially bypassing your EQ or something, though this is an issue seen by others without fancy audio setups as well).

I do not know what it is (and WV2 being closed-source, it could be something entirely unfixable on my end), which makes it difficult to track down and is why this issue hasn't really moved along. I do appreciate your willingness to help debug though!

**Comment by 1111-glitch** [comment]
That's unfortunate :( Thank you for the responses regardless and for making such a great piece of software. I've gotten multiple of my super nerdy efficiency oriented friends to use it, especially since Discord seems to break horribly with 8k polling mice.
